packages/commerce-server/src/record-new-purchase.ts

```diff
@@ -99,52 +88,52 @@ export async function recordNewPurchase(
     getMerchantProduct,
   } = getSdk()
 
-  const {stripeCtx} = options
-
-  const purchaseInfo = await stripeData({checkoutSessionId, stripeCtx})
+  const purchaseInfo = await options.paymentProvider.getPurchaseInfo(
```

Comment:
using the payment provider in here to `getPurchaseInfo`, though you'll notice above that the `stripeData` function still exists. that is because many of the apps still rely on it. once I've configured payment providers in each of those apps, they will be able to switch over to the new implementation and we can ditch `stripeData`.